### ISSUE TITLE
multiple domains in a project

### ISSUE THREAD

**Issue by mislavcimpersak** [chronark/terraform-provider-vercel#42]
Hi there!

Thank you for creating this, it's really a great terraform provider for a great platform!

Currently I'm able to set a single domain with a single git branch per project, but often in reality we need multiple domains.

```
resource "vercel_project" "web" {
  name    = "web"

  git_repository {
    type = "gitlab"
    repo = "foo/bar"
  }

  domain {
    name = "foo-dev.example.com"
    git_branch = "master"
  }
}
```

Vercel UI and API allow for multiple domains in a single project. Using the API we could set up one domain for development, one for staging etc. Something like this:

```
resource "vercel_project" "web" {
  name    = "web"

  git_repository {
    type = "gitlab"
    repo = "foo/bar"
  }

  domain {
    name = "foo-dev.example.com"
    git_branch = "master"
  }

  domain {
    name = "foo-stage.example.com"
    git_branch = "stage"
  }
}
```

Currently, [I see](https://github.com/chronark/terraform-provider-vercel/blob/363ea8c4570cad1daadf28ecaa0be3db2eb9f6dc/internal/provider/resource_project.go#L65) that you are limiting the number of `domain` blocks to 1 which is why the example above is (expectedly) failing.

---

Temp workaround could be to create _N_ number of projects where _N_ is the number of domains and their appropriate deploy environments.

**Comment by chronark** [comment]
Hey @mislavcimpersak 
Fortunately recurser has implemented the necessary changes for this in #38 recently.

I have released `v0.13.0-rc.1` with the ability to define multiple `domain` blocks per project to try it out

**Comment by chronark** [comment]
Seems to work fine on my end, so I released it as `v0.13.0`.

**Comment by mislavcimpersak** [comment]
Confirming that it is working. Thank you @chronark 💯 

Closing the issue.
